handler/metadata.go

```diff
@@ -0,0 +1,74 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/ONSdigital/dp-population-types-api/config"
+	"github.com/ONSdigital/dp-population-types-api/contract"
+	"github.com/ONSdigital/dp-population-types-api/datastore"
+	"github.com/go-chi/chi/v5"
+	"github.com/pkg/errors"
+)
+
+type Metadata struct {
+	cfg         *config.Config
+	respond     responder
+	MongoClient Datastore
+}
+
+// NewMetada returns a new Metadata Handler
+func NewMetada(cfg *config.Config, r responder, databaseClient Datastore) *Metadata {
```

Comment:
Any reason you couldn't use `d` for the variable name to keep with the convention?